### PR TITLE
Removing "manoto" target and using MANOTO=1 env variable instead.

### PR DESCRIPTION
**Pull request by xiam**
Related: https://github.com/getlantern/lantern/pull/2871#issuecomment-127761102

### CODE REVIEW

**Comment by xiam** [comment]
@myleshorton could you check this one for #2871? Thanks!

File: installer-resources/windows/.packaged-lantern.yaml [hunk 1]
```diff
@@ -1 +1 @@
-
+startupurl: https://www.facebook.com/manototv
```
Comment (myleshorton) [inline installer-resources/windows/.packaged-lantern.yaml]: I don't think we want this set by default

**Comment by myleshorton** [comment]
Other than the manoto url in the installer-resources/windows/.packaged-lantern.yaml file, this looks great to me @xiam!

File: installer-resources/windows/.packaged-lantern.yaml (hunk 1 above)
Comment (xiam) [inline installer-resources/windows/.packaged-lantern.yaml]: Oh no, this was commited by error. I actually think we can safely remove those .packaged-lantern.yaml files and add them to the ignore list, as they're generated by the Makefile.

Comment (myleshorton) [inline installer-resources/windows/.packaged-lantern.yaml]: That's fine too.

**Comment by myleshorton** [comment]
I just tested this on all platforms, and it's working great.

**Comment by xiam** [comment]
Thanks @myleshorton. Everything should be set now.

**Comment by myleshorton** [comment]
OK merging!

**Pull request merged**